system_test/common/helpers/aest_docker.erl

```diff
@@ -237,10 +239,14 @@ setup_node(Spec, BackendState) ->
     Context = #{aeternity_config => RootVars},
     {ok, ConfigString} = write_template(TemplateFile, ConfigFileHostPath, Context),
     Command =
-        case MineRate of
-            default -> [];
-            _ when is_integer(MineRate), MineRate > 0 ->
-                ["-aecore", "expected_mine_rate", MineRate]
+        case maps:find(custom_command, Spec) of
```

Comment:
Could you please document this `custom_command` optional spec item in `type node_spec()`, also adding comment on how semantics of `custom_command` and `mine_rate` are meant to interact?